pkg/testutils/testutils.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2015 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package testutils
```

Comment:
Let's put this into pkg/db. syz-stress is not a test utility. And having a single function in a package is generally a wrong granularity.